hw/opentitan/ot_spi_device.c

```diff
@@ -840,10 +840,15 @@ static void ot_spi_device_release(OtSPIDeviceState *s)
          * "does not show the commands falling into the mailbox region or
          *  Read SFDP command’s address."
          */
-        if (ot_spi_device_is_hw_read_command(s) &&
-            !ot_spi_device_is_mailbox_match(s, f->address)) {
-            trace_ot_spi_device_update_last_read_addr(s->ot_id, f->address);
-            s->spi_regs[R_LAST_READ_ADDR] = f->address;
+        if (ot_spi_device_is_hw_read_command(s)) {
+            // Address has already been incremented by read commands.
```

Comment:
Nit: should be `/* */` style comment.


```suggestion
            /* address has already been incremented by read commands */
```

Reply:
Sorry, I do this every time